### PR TITLE
test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
- [DEMO LINK](https://OleksandrFEdev
/
layout_stop-watch.github.io/layout_stop-watch/)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://OleksandrFEdev
+/
+layout_stop-watch.github.io/layout_stop-watch/)
```
Comment (vlad-demchuk) [inline readme.md:5]: fix demo link

File: src/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  padding: 0;
+}
+
+body {
+  height: 100vh;
+  background: linear-gradient(12.3deg, rgb(129, 238, 129),
+  rgb(255, 217, 0), rgb(121, 21, 21));
+}
+
+.watch {
```
Comment (vlad-demchuk) [inline src/styles/main.scss:13]: create separate files per bem blocks

File: src/styles/main.scss [hunk 3]
```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```
Comment (vlad-demchuk) [inline src/styles/main.scss:4]: do not use `*`  selector to reset paddings or margins

use some specific tags for it

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/styles/main.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  background: linear-gradient(12.3deg, rgb(129, 238, 129),
+  rgb(255, 217, 0), rgb(121, 21, 21));
+}
+
+@import "./watch.scss";
```
Comment (anastasiiavorobiova) [inline src/styles/main.scss:10]: It's a good practice to import partials containing keyframes and other global styles first.

**Review by anastasiiavorobiova — COMMENTED** [review]
Unfortunately, I couldn't open your demo link.

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Unfortunately, I couldn't open your demo link

File: readme.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Stopwatch task
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stop-watch/)
+- [DEMO LINK](https://OleksandrFEdev /layout_stop-watch.github.io/layout_stop-watch/)
```
Comment (anastasiiavorobiova) [inline readme.md:3]: fix the demo link to https://YOUR_GITHUB_ACCOUNT_NAME.github.io/DEPLOYED_FOLDER_NAME

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Couldn't open the demo link

File: readme.md (hunk 5 above)
Comment (OleksandrFEdev) [reply readme.md]: - [DEMO LINK](https://OleksandrFEdev.github.io/layout_stop-watch/)

File: src/styles/watch.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  background: linear-gradient(229deg, #fbff00, #adaa16, #c7d413);
+  background-size: 240% 240%;
+  animation: gradient-animation 16s ease infinite;
+}
+
+.watch__arrows-minute {
+  height: 200px;
+  width: 18px;
+  background-color: #464861;
+  animation: hand-rotate 3600s infinite steps(60);
```
Comment (polosanya) [inline src/styles/watch.scss:31]: It would be great to create variables for all main values (sizes, animation duration etc)

**Review by polosanya — CHANGES_REQUESTED** [review]
The center dot should be over the other elements. Change z-index property
image

**Review by nazarmatsevych — APPROVED** [review]
👍

**Pull request closed without merging**